simapp/encoding.go

```diff
@@ -17,3 +18,12 @@ func MakeTestEncodingConfig() simappparams.EncodingConfig {
 	ModuleBasics.RegisterInterfaces(encodingConfig.InterfaceRegistry)
 	return encodingConfig
 }
+
+// MakeTestCodecs is a helper function which uses MakeTestEncodingConfig to
+// construct new *std.Codec and *codec.LegacyAmino instances.
+// It must not be used in non test files, instead app.AppCodec should be used.
+// [DEPRECATED]
+func MakeTestCodecs() (codec.Marshaler, *codec.LegacyAmino) {
```

Comment:
Can we just remove `MakeTestCodecs` completely? It just takes a subset of `MakeTestEncodingConfig`, so we could just use that instead.

Reply:
I was thinking about it when doing this PR, but wanted to limit amount of changes. I like the Idea.